rtublen/Studentportalen
# Add students to start page

# Description

Adds students to start page.

Fixes # US001

## Type of change

Please delete options that are not relevant.


# How to verify change

1. Start application
2. Verify that students are listed on start page.

# Checklist:
